orchestrator/modules/tools/tool_router.py

```
@@ -567,6 +567,56 @@ class ToolRouter:
+            from core.database.database import SessionLocal
+            from modules.rag.retrieval_filters import build_retrieval_filters, allowed_document_ids
+            from modules.tools.discovery.handlers_documents import _resolve_agent_team
+
+            db = SessionLocal()
+            try:
+                team = _resolve_agent_team(db, agent_id)
+                filters = build_retrieval_filters(workspace_id=str(workspace_id), team=team)
+                allowed = allowed_document_ids(db, doc_ids, filters)
+            finally:
+                db.close()
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)